lms/djangoapps/bulk_email/migrations/0004_seed_aws_flags.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+from django.db import migrations, models
+
+def forwards_func(apps, schema_editor):
+    if os.environ["DJANGO_SETTINGS_MODULE"] == "lms.envs.aws":
```

Comment:
Verdict: option 1, we'll just take it down for a short time and get the flag turned on ASAP after deployment.